src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ info.app.version=${version:0.0.1}
 server.port=8087
 
 spring.datasource.url=jdbc:postgresql://${db.host:localhost}:${db.port:5432}/${db.name:postgres}?currentSchema=${db.schema.name:cs_audit_service}&user=${db.username:root}&password=${db.password:dev}&stringtype=unspecified
-
+spring.flyway.schemas=${db.schema.name:cs_audit_service}
```

Comment:
Please leave this setting unset unless you are 100 sure it was always set to that value as a default. Otherwise we run a risk of breaking prod:D